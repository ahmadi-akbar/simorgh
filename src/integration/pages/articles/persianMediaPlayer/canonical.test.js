/**
 * @service persian
 * @pathname /persian/articles/cej3lzd5e0go
 */

import runCanonicalTests from '../canonicalTests';
import runMediaPlayerCaptionTests from '../mediaPlayerCaption';
import runMediaPlayerPlaceholderTests from '../mediaPlayerPlaceholderTestsCanonical';

describe('Canonical', () => {
  describe(pageType, () => {
    runCanonicalTests(service);
    runMediaPlayerCaptionTests();
    runMediaPlayerPlaceholderTests();
  });
});
