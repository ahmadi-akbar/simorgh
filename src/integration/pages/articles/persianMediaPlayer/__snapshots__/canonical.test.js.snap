// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Canonical Articles Media Player Caption should have a caption under the media player 1`] = `"توضیح ویدئو، این تصویر آزمایشی ، کپی رایت بی بی سی ، نقشه ای از فرانسه را نشان می دهد. تصویر در سه بلوک اول است و این عنوان را دارد."`;
